swap_meet/vendor.py

```diff
@@ -1,2 +1,105 @@
 class Vendor:
-    pass
+
+    def __init__(self, inventory=None):
+        self.inventory = inventory if inventory else []
+
+    def add(self, new_item):
+        """ Adds argument passed into new_item to Vendor's inventory list, and returns the new item. """
+        self.inventory.append(new_item)
+        return new_item
+
+    def remove(self, item_to_remove):
+        """ Removes argument passed into item_to_remove from Vendor's inventory list, and returns the removed item. """
+        if item_to_remove not in self.inventory:
+            return False
+        self.inventory.remove(item_to_remove)
+        removed_item = item_to_remove
+        return removed_item
```

Comment:
Nice guard clause!

Reply:
The value of `item_to_remove` isn't mutated/changed or removed from memory so there's no need to create another variable to represent it. The `remove` method only removes an element from the list that matches that value of `item_to_remove`. We can return the variable as is. 

```suggestion
    def remove(self, item_to_remove):
        """ Removes argument passed into item_to_remove from Vendor's inventory list, and returns the removed item. """
        if item_to_remove not in self.inventory:
            return False
        self.inventory.remove(item_to_remove)
        return item_to_remove
```